src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java

```diff
@@ -25,8 +25,8 @@
 
 import java.util.*;
 
-import static org.broadinstitute.hellbender.utils.GATKProtectedVariantContextUtils.getAttributeAsLong;
-import static org.broadinstitute.hellbender.utils.GATKProtectedVariantContextUtils.getAttributeAsLongList;
+import static org.broadinstitute.hellbender.utils.variant.VariantContextGetters.getAttributeAsLong;
```

Comment:
We typically don't use static imports at all.

Reply:
I blame IntelliJ.

Reply:
fixed it